src/EFCore.PG.NTS/EFCore.PG.NTS.csproj

```diff
@@ -29,4 +29,7 @@
     <PackageReference Include="Npgsql.NetTopologySuite" Version="1.0.1" />
     <PackageReference Include="Npgsql" Version="4.0.0" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="SourceLink.Create.CommandLine" Version="2.8.2" />
+  </ItemGroup>
```

Comment:
Isn't this already be included via Directory.Build.props? What's the benefit of including it in each `.csproj`?

https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/blob/84ecb43fc106e06450a5dbf7d1884eb2924070b1/src/Directory.Build.props#L1-L5